test/configCases/plugins/environment-plugin/index.js

```diff
@@ -33,3 +33,17 @@ it("should import process.env var with empty value", () => {
 	if(process.env.III !== "")
 		require.include("iii");
 });
+
+it("should import process.env var by process.env", () => {
+	if(process.env.IIII !== "iiii") {
+		if (parseInt(process.version.match(/^v(\d+).*$/i)[1]) > 4) {
+			eval("let {IIII} = process.env;");
+		} else {
+			var env = process.env;
+			var IIII = env.IIII;
+		}
+		if(IIII !== "iiii") {
+			require.include("iiii");
+		}
```

Comment:
try something like

``` js
var { TEST } = process.env;
if(TEST !== "ok")
  require.include("fail");
```

but put it into a separate test case which is guarded by a `test.filter.js` file.

Also check if it's possible to use multiple `EnvironmentPlugin`s.